### ISSUE TITLE
Batch norm and ReLU reversed?

### ISSUE THREAD

**Issue by urielka** [Zehaos/MobileNet#7]
Hi according to the paper (and as you posted in the README) the basic building block is:
<img width="336" src="https://cloud.githubusercontent.com/assets/30956/25765762/46c7a8c8-31f7-11e7-93d9-0f64eb95809b.png">

But if I follow TF-Slim code and your code I think you are having depthwise -> ReLU -> BN -> Pointwise -> ReLU -> BN.

The ReLU is part of slim convolution layers.

Also you seem to be using separable_convolution2d with num_outputs=None to get a depthwise convolution, you can just use https://www.tensorflow.org/api_docs/python/tf/nn/depthwise_conv2d instead, no?

**Comment by Zehaos** [comment]
Hi,
1. please refer to slim arg scope mechanism. In my implementation ReLU is the activation function of BN. 
2. tf.nn.depwisw_conv work in the same way, but it needs additional variable creation.

**Comment by urielka** [comment]
Understood, my bad :)
